tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -551,11 +567,12 @@ EOT
 	AGENT_DIR="${ROOTFS_DIR}/usr/bin"
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
+	if [ "$ARCH" == "ppc64le" ] || [ "$ARCH" == "s390x" ]; then
+		LIBC=gnu
+		echo "WARNING: Forcing LIBC=gnu because $ARCH has no musl Rust target"
```

Comment:
Nit: `>&2`?

Reply:
yeah, I only moved this, but might be a good opportunity to do that